contracts/storage/ProfileStorage.sol

```diff
@@ -49,7 +54,66 @@ contract ProfileStorage {
         _;
     }
 
+    function createProfile(
+        address operationalWallet,
+        address managementWallet,
+        bytes nodeId,
+        uint96 initialAsk,
+        uint96 initialStake
+    )
+        public
+        OnlyContracts
+        returns (uint96, address)
+    {
+        require(!identityIds[operationalWallet] && !identityIds[managementWallet], "Profile already exists");
+        require(!nodeIdsList[nodeId], "Node ID connected with another profile");
+        require(nodeId.length != 0, "Node ID can't be empty");
+        require(initialAsk > 0, "Ask can't be 0");
+
+        Identity identity = new Identity(operationalWallet, managementWallet);
+        address identityContractAddress = address(identity);
+
+        ProfileDefinition memory profile = ProfileDefinition({
+            ask: initialAsk,
+            stake: initialStake,
+            reward: 0,
```

Comment:
General comment - I believe this structure is OK for this release, yet for the next one we introduce eras for the reward (for token delegation)